SpatialGDK/Documentation/content/tutorials/multiserver-shooter/tutorial-multiserver-healthchanges.md

```diff
@@ -0,0 +1,72 @@
+<%(TOC)%>
+# Multiserver Shooter Tutorial
+
+## Step 2: Replicate health changes
+
+In this project each `TPSCharacter` contains a variable called `CurrentHealth`, which keeps track of that character's health. On your servers, `CurrentHealth` is reduced whenever a character is shot, but this reduction is not replicated on the clients connected to the game. This is because the `CurrentHealth` variable is not setup for replication.
```

Comment:
```suggestion
In this project each `TPSCharacter` contains a variable called `CurrentHealth`, which keeps track of that character's health. On your servers, `CurrentHealth` is reduced whenever a character is shot, but this reduction is not replicated on the clients connected to the game. This is because the `CurrentHealth` variable is not set up for replication.
```